src/server.c

```diff
@@ -5200,10 +5203,11 @@ void setupChildSignalHandlers(void) {
  * of the parent process, e.g. fd(socket or flock) etc.
  * should close the resources not used by the child process, so that if the
  * parent restarts it can bind/lock despite the child possibly still running. */
-void closeClildUnusedResourceAfterFork() {
+void closeChildUnusedResourceAfterFork() {
     closeListeningSockets(0);
-    if (server.cluster_enabled && server.cluster_config_file_lock_fd != -1)
-        close(server.cluster_config_file_lock_fd);  /* don't care if this fails */
+
+    if (server.cluster_enabled && server.cluster_config_file_lock_fd != -1) 
+        close(server.cluster_config_file_lock_fd);  /* don't care if this fails */ 
```

Comment:
looks like you added a trailing white space here.

Reply:
Fixed.